### ISSUE TITLE
Write more docstrings and comments

### ISSUE THREAD

**Issue by felipedau** [AnemoneLabs/unmessage#82]
This has been requested by @david415. unMessage initially had more docstrings and comments, but that habit got lost :/ Some ideas for this:

- I think that #66 should be finished first, because the public API will change
- Comment what is necessary - good code is mostly self explanatory
- If a comment is long, simplify the code
- Ask someone that is not familiar to the code to read it - improve based on their feedback
- Create docs for the public API (this can also be a separate issue)
